speechbrain/utils/parameter_transfer.py

```diff
@@ -181,12 +186,7 @@ def split(src):
         else:
             return split(path)
 
-    def collect_files(
-        self,
-        default_source=None,
-        use_auth_token=False,
-        local_strategy: LocalStrategy = LocalStrategy.SYMLINK,
-    ):
+    def collect_files(self, default_source=None, fetch_config=FetchConfig()):
```

Comment:
Wondering about this. `fetch_config=FetchConfig()` is evaluated once at function definition time, so every call to this function that doesn't pass a `fetch_config` will reuse the same instance. So, if one call for whatever reason is modifying one attribute of `FetchConfig`, then it will backpropagate to all `collect_files` functions. Wouldn't it be better to have this parameter set to `None`, and then in the function do something like:
```
if fetch_config is None:
   fetch_config = FetchConfig()
```

Reply:
Good catch.

*turns toward void and inhales deeply* CURSE YOU PYTHON AND YOUR FRIGGEN UNINTUITIVE DEFAULT PARAMTERS